examples/dt/compute/amd-sev/readme.md

```diff
@@ -0,0 +1,62 @@
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change              |  Date   |
+| -------: | :------------------ | :-----: |
+|     v0.1 | Initial publication | 2023-12-14 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | bm    | 2      |
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+| Compute   | set-nova-scheduler-filter    | Must have          |
+
+
+## Support services
+## Additional services required for integration testing that may not be the subject of this DT
+| Service   |  Reason                |
+| --------- | ---------------------- |
+| Glance    | Must have              |
+| Keystone  | needed by all services |
+| FIPS      | Enabled by default     |
+
+#### Nova
+| Service         | configuration |
+| ----------------| --------------|
+| Image_type      | qcow          |
```

Comment:
* qcow2 is our defaults https://github.com/openstack-k8s-operators/nova-operator/blob/main/templates/nova.conf#L183 
* What configuration we need to mention here? E.g. you set machine_type to q35 above. Do we need to mention it here? btw q35 is also in our default now https://github.com/openstack-k8s-operators/nova-operator/blob/main/templates/nova.conf#L179

Reply:
ack